packages/core/DEVELOPMENT.md

```diff
@@ -1,92 +1,40 @@
 # Development
 
-## Development environment
+### Prerequisites
 
-> Helsinki Design System uses [**Lerna**](https://lerna.js.org/) for running scripts across the repo as well as versioning and creating releases of the packages. [**Yarn workspaces**](https://yarnpkg.com/lang/en/docs/workspaces/) is used to manage dependencies. This allows the separate packages to reference each other via symlinks during local development.
+Set up your local development environment by following the steps in the [development guide](../../DEVELOPMENT.md#setting-up-local-development-environment)
 
 ### Setting up local development environment
 
-1. Clone the HDS repository.
-```bash
-git clone https://github.com/City-of-Helsinki/helsinki-design-system.git
-```
+1. Build the design tokens.
 
-2. Go to the root of the project and install dependencies with `yarn`.
-```bash
-cd helsinki-design-system
-yarn
-```
-
-3. Build the design tokens.
 ```bash
 yarn build:tokens
```

Comment:
I think it should still be specified that these commands (yarn build:tokens and yarn start:core) are run from the root folder. I can already envision users running these commands from the package folder and being all confused when they get errors. It might be enough to add that information on the first step, like "1. Build the design tokens in the root folder." ? Or have it as the first step, or even before the steps as "Go through these steps in the root of the project."

If you think this is a good fix, it should be done in similar fashion in the other packages' DEVELOPMENT.md files as well.

Reply:
I was thinking this too, I'll update for all DEVELOPMENT.md files.